example/cumulus-tf/variables.tf

```diff
@@ -1,5 +1,11 @@
 # Required
 
+variable "cloudwatch_log_retention_periods"{
+  type = map(string)
```

Comment:
Should this be a map of strings? Seems like the retention periods may need to be a map of numbers

Reply:
```suggestion
  type = map(number)
```